scripts/apps/authoring/macros/macros.ts

```diff
@@ -352,7 +352,16 @@ function MacrosReplaceDirective(editorResolver) {
                 return scope.diff[from] || null;
             }
 
-            init(scope.diff);
+            // There may be multiple instances of editors. Try waiting for all.
+            const initializeMacros = debounce(once(() => {
+                init(scope.diff);
```

Comment:
not sure this is the case, but you might need to call `scope.apply();` here to make it re-render anything angular:
```ts
scope.apply(() => init(scope.diff));
```